From 33533c4d63ab7b323c3f03def90981163e5dbfc9 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:00:16 -0400
Subject: [fs] btrfs: sysfs: list the NO_HOLES feature

Message-id: <1407179109-13714-130-git-send-email-sandeen@redhat.com>
Patchwork-id: 87075
O-Subject: [RHEL7.1 PATCH 129/422] btrfs: sysfs: list the NO_HOLES feature
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: David Sterba <dsterba@suse.cz>

commit c736c095de53c1a0a23909239d69bb56693df8ef
Author: David Sterba <dsterba@suse.cz>
Date:   Tue Jan 21 18:56:09 2014 +0100

    btrfs: sysfs: list the NO_HOLES feature

    Signed-off-by: David Sterba <dsterba@suse.cz>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/sysfs.c b/fs/btrfs/sysfs.c
index 1a89386..782374d 100644
--- a/fs/btrfs/sysfs.c
+++ b/fs/btrfs/sysfs.c
@@ -202,6 +202,7 @@ BTRFS_FEAT_ATTR_INCOMPAT(big_metadata, BIG_METADATA);
 BTRFS_FEAT_ATTR_INCOMPAT(extended_iref, EXTENDED_IREF);
 BTRFS_FEAT_ATTR_INCOMPAT(raid56, RAID56);
 BTRFS_FEAT_ATTR_INCOMPAT(skinny_metadata, SKINNY_METADATA);
+BTRFS_FEAT_ATTR_INCOMPAT(no_holes, NO_HOLES);
 
 static struct attribute *btrfs_supported_feature_attrs[] = {
  BTRFS_FEAT_ATTR_PTR(mixed_backref),
@@ -212,6 +213,7 @@ static struct attribute *btrfs_supported_feature_attrs[] = {
  BTRFS_FEAT_ATTR_PTR(extended_iref),
  BTRFS_FEAT_ATTR_PTR(raid56),
  BTRFS_FEAT_ATTR_PTR(skinny_metadata),
+ BTRFS_FEAT_ATTR_PTR(no_holes),
  NULL
 };
 
-- 
1.7.1